packages/contracts/contracts/deploy/IPayableDAppFactory.sol

```diff
@@ -37,6 +37,10 @@ interface IPayableDAppFactory {
         uint256 _runway
     ) external;
 
+    /// @notice sets a name for the factory using ENS service
+    /// @param _name name of the factory
```

Comment:
We usually capitalize the first word after `@<tag>`, because, if this text gets scrapped to generate documentation, it will be converted into an HTML paragraph. Also, we generally use definite articles to describe parameters.

```suggestion
    /// @notice Sets a name for the factory using ENS service
    /// @param _name The name of the factory
```